rasa/utils/plotting.py

```diff
@@ -192,9 +192,11 @@ def _extract_paired_histogram_specification(
     histograms = [
         # A list of counts - how often a value in `data` falls into a particular bin
         # [numpy-upgrade] type ignore can be removed after upgrading to numpy 1.23
```

Comment:
can this also be removed?

Reply:
yes I removed all of these now, since I upgraded to 1.23.x